packages/design-system/src/components/Dropdown/Dropdown.tsx

```diff
@@ -280,6 +283,7 @@ export const Dropdown: React.FC<DropdownProps> = (props: DropdownProps) => {
               title="selected option icon"
               viewBox="0 0 448 512"
               className="ds-u-font-size--sm"
+              id={selectedIconId}
             >
               {checkIcon}
```

Comment:
it's a shame we didn't name this `{selectedIcon}` for consistency. i like the id being `{selectedIconId}` because it states what purpose it serves (icon for what's selected).

Reply:
I could change the `checkIcon` name